index.html

```diff
@@ -0,0 +1,191 @@
+                     //The dialog that comes up on the alert box
+                     .textContent('If you haven\'t fill out some boxes please do so!\n If you have just sit back and enjoy the event!')
+                     .ariaLabel('Thank You')
+                     //The close button for the alert box
+                     .ok('Ok!')
+                     .targetEvent(ev)
+               );
+            };           
+         }                 
+      </script>     	  
+   </head>
+   <body ng-app="signIn">
+      <div id="title" class="introText">
+	      <h1> ACM Sign-In </h1>
+      </div>
+
+	  
+      <div id="inputContainer" class="completeCard" ng-controller="inputController as ctrl" ng-cloak>
+         <md-content layout-padding>
+			     <md-card md-theme="{{ showDarkTheme ? 'dark-grey' : 'default' }}" md-theme-watch="">
+              <md-card-title>
+                <md-card-title-text>
+                  <span class="md-headline">Login</span>
+			
+	            		<form method="post" name="projectForm">  
+                    <md-input-container>
+                      <label>Secret Key</label>
+                      <input required name="key" ng-model="project.key" value="{name}">
+                      <md-tooltip>Written on a board or ask an ACM representative</md-tooltip>
+                      <div ng-messages ="projectForm.key.$error">
+                        <div ng-message = "required"> This is required </div>
+                      </div>
+                    </md-input-container>
+                    <md-input-container class="md-block">
+                      <label>Name</label>
+                      <md-tooltip>First and Last Name</md-tooltip>                  
+                      <input required name="name" ng-model="project.name" value="{email}">
+                      <div ng-messages="projectForm.userName.$error">
+                        <div ng-message="required">This is required.</div>
+                      </div>
+                    </md-input-container>
+                    
+                    <md-input-container class="md-block">
+                      <label>Email</label>
+                      <input required type="email" name="userEmail" ng-model="project.userEmail"
+                       minlength="5" maxlength="100" ng-pattern="/^.+@.+\..+$/" /  value = "{major}">
+                      <md-tooltip>Please use your Missouri S&T Email</md-tooltip>
+                      <div ng-messages="projectForm.userEmail.$error" role="alert">
+                       <div ng-message-exp="['required', 'minlength', 'maxlength', 'pattern']">
+                        Your email must be between 5 and 100 characters long and should be a valid email address.
+                       </div>
+                      </div> 
+                    </md-input-container>  
+                
+                    <md-input-container-select>
+                      <md-tooltip>All Majors are in Alphabetical Order</md-tooltip>
```

Comment:
There will be people attending who are not students, so this needs an option for them too.